src/Engine/MapController.java

```diff
@@ -204,20 +204,22 @@ private void handleFreeTileSelect(InputHandler.InputAction input)
         shouldConsider = false;
         break;
       case BACK:
-        myGameInputHandler.back();
-        shouldConsider = false;
-
-        // If we hit BACK while over a unit, add it to the threat overlay for this CO
-        Location loc = myGame.gameMap.getLocation(myGame.getCursorCoord());
-        Unit resident = loc.getResident();
-        if( null != resident )
+        if( !myGameInputHandler.isTargeting() )
         {
-          ArrayList<Unit> threats = myGame.activeCO.threatsToOverlay;
-          if( threats.contains(resident) )
-            threats.remove(resident);
-          else
-            threats.add(resident);
+          // If we hit BACK while over a unit, add it to the threat overlay for this CO
+          Location loc = myGame.gameMap.getLocation(myGame.getCursorCoord());
+          Unit resident = loc.getResident();
+          if( null != resident )
+          {
+            ArrayList<Unit> threats = myGame.activeCO.threatsToOverlay;
+            if( threats.contains(resident) )
+              threats.remove(resident);
+            else
+              threats.add(resident);
+          }
         }
+        myGameInputHandler.back();
```

Comment:
This used to only happen `if( myGameInputHandler.isTargeting() )`. Why the change?

Reply:
I reversed that because it brought back the "unit is now very excited" after you click on it thing.
I might be able to remove it entirely in the `transaction` PR, though.

Reply:
Does it? I don't see that.

Reply:
I apparently forgot to push it up. Whoops.
Bah, I need to fiddle with stuff a bit.

Reply:
Okay, after having doublechecked...
I think I can move it back if necessary, but we can't get rid of it entirely or you can't back out of launching flares.